Python/Network_scanner/readme.md

```diff
@@ -0,0 +1,32 @@
+
+# Network Scanner in Python
+Python script that can scan all networks connected to same router and give's thier respectieve IP and MAC address.
+
+# Packages Used
+1. Scapy
+2. Optparse
+
+# How to run
+**1.** Fork [this](https://github.com/HarshCasper/Rotten-Scripts/) repository.
+Click on the <a href="https://github.com/HarshCasper/Rotten-Scripts/"><img src="https://img.icons8.com/ios/24/000000/code-fork.png"></a> symbol at the top right corner.
+
+**2.** Clone the forked repository.
+
+```bash
+git clone https://github.com/<your-github-username>/Rotten-Scripts
+```
+**3.** Navigate to the project directory.
+
+```bash
+cd Rotten-Scripts/python/Network_Scanner
+```
+```bash
+python network_scanner.py
+```
+## Output
+
+![](https://github.com/pritamp17/ethical_hacking_tools/blob/main/FILE_INTERCEPTOR/Linux.png?raw=true)
+![](https://github.com/pritamp17/ethical_hacking_tools/blob/main/FILE_INTERCEPTOR/windows.png?raw=true)
```

Comment:
These references won't work if you delete your Fork. Please upload the Images on Imgur and reference them from the same.